src/group.coffee

```diff
@@ -219,18 +235,22 @@ module.exports = (robot) ->
       return
     response = []
     for name in names
-      if group.remove g, name
-        response.push "#{name} removed from group #{g}."
-      else
-        response.push "#{name} is not in group #{g}!"
+      for k, u of robot.brain.data.users
+        if u.name == name
+          if group.remove g, u.id
+            response.push "#{name} removed from group #{g}."
+          else if group.remove g, name
```

Comment:
Why would name be in `g` at all? I think we can remove this case, right?

Reply:
On second thought, I realize that the two things I commented on were both for backwards compatibility. Do you think it's better if we just remove them and release as 2.x, or do you think we should stick with a backwards-compatible release?

I think it's better to err on the side of 2.x. I don't know how your new changes interact with non-Slack chat applications, so it would be better not to disturb existing working installations of 1.x with a new release. And in that case, we might as well remove the backwards compatibility code and start fresh.